api/legacy/v1/groupversion_info.go

```diff
@@ -1,6 +1,6 @@
 // Package v1 contains API Schema definitions for the topolvm v1 API group
-//+kubebuilder:object:generate=true
-//+groupName=topolvm.cybozu.com
+// +kubebuilder:object:generate=true
```

Comment:
These comments are used by kubebuilder. I'm not sure if it will be recognized correctly, so please do not change the spaces.

Reply:
The above code generation results from adjustments made by `go fmt`. It does not seem to affect the behavior of kubebuild even if it contains blanks (ref: https://github.com/golang/go/issues/54349). The difference between the result of `go fmt` and that of kubebuilder will be solved by fixing [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v4/scaffolds/internal/templates/api/group.go#L57).